Lib/gftools/builder/schema.py

```diff
@@ -0,0 +1,55 @@
+from strictyaml import load, Map, MapPattern, Str, Int, Seq, YAMLError, Optional, Bool
```

Comment:
Wrap these in parenthesis 

```
from strictyaml (
    load,
    Map,
    ...
)
```

Include a one sentence doc string as well.

Reply:
or just run black on the file.

Reply:
Thanks. This is fixed in commit 9463326 that I just pushed to this PR. Black `v20.8b1` doesn't format it this way for me, for some reason.